.lycheeignore

```diff
@@ -30,6 +33,13 @@ https://www.aftership.com/
 https://www.digikey.se/product-detail/en/jst-sales-america-inc/PHR-2/455-1165-ND/608607
 https://www.java.com/en/download/manual.jsp
 https://www.amazon.com/gp/help/customer/display.html*
+https://se.mathworks.com/store*
+https://se.mathworks.com/support/contact_us.html
+https://se.mathworks.com/matlabcentral/answers/index
+https://se.mathworks.com/help/install/install-products.html
+https://se.mathworks.com/company/aboutus/contact_us/arduino_engineering_kit.html
+https://se.mathworks.com/campaigns/products/arduino-kit-rev2-license.html
```

Comment:
Try to avoid lanugage/locale codes from URLs, like the **se** subdomain here.

Usually, they can be removed, and we'll rely on the site to select the right locale for the user.

If a locale has to be specified, use US/English.

Please try removing "se." and verify that the URLs still work (they may still redirect to the "se." pages based on IP or cookies – that's fine) and make the corresponding changes in the articles.